uktrade/data-hub-frontend
# Fix/edit interaction styling

## Description of change

Fixes a misaligned pipeline graphic next to a sub-select in the edit interaction form's Service section.

## Test instructions

1. Go to `/interactions/<id>/edit`
2. Select a value from the select field right below the _Service_ heading (yes, the field doesn't have a label), other than _Other advice and information_
3. There should appear a sub-select (again, no label)
4. The sub field should have an L-shaped pipeline graphic to its left leading from the select above.
5. The pipeline should be nicely aligned so, that it leads to the vertical middle of the sub-select
6. The carret on the very right of the sub-field should stay aligned in the same way as in other selects at all times

## Screenshots

### Before

<img width="1045" alt="Screenshot 2024-09-27 at 15 35 01" src="https://github.com/user-attachments/assets/6249320c-3830-4ef1-8437-89a7c98921c4">

### After
<img width="1150" alt="Screenshot 2024-10-01 at 17 22 23" src="https://github.com/user-attachments/assets/3082abde-0d36-4d5d-9eab-75e2c22d77d7">
